modules/gitops-argocd-cli-enabling-tab-completion.adoc

```diff
@@ -0,0 +1,41 @@
+// Module is included in the following assemblies:
+//
+// * gitops_cli_argocd/configuring-argocd-gitops-cli.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-argocd-cli-enabling-tab-completion_{context}"]
+= Enabling tab completion
+
+After you install the {gitops-shortname} `argocd` CLI, you can enable tab completion to automatically complete `argocd` commands or suggest options when you press Tab.
+
+[NOTE]
+====
+Tab completions only exist for the Bash shell.
```

Comment:
I didn't notice this on my first review, but IIRC, it also exists for Zsh.

I think it's fine that the procedure only covers the Bash shell, as that's the most common shell _by far_ FWIW.